test/modules/geo-layers/mvt-layer.spec.js

```diff
@@ -282,30 +297,39 @@ test('MVT dataInWGS84', async t => {
     zoom: 0
   });
 
+  const onAfterUpdate = ({layer}) => {
+    if (layer.isLoaded) {
+      const tile = layer.state.tileset.selectedTiles[0];
+      const contentWGS84 = tile.dataInWGS84;
+
+      t.deepEqual(
+        layer.props.binary ? contentWGS84 : contentWGS84[0].geometry.coordinates,
```

Comment:
Rather than adapt the test we need to make dataInWGS84 returns the same in binary vs non-binary

tile.dataInWGS84 should always return an array of geojson features